uast/ann/ann.go

```diff
@@ -53,14 +53,20 @@ type Rule struct {
 	rules      []*Rule
 }
 
+const (
```

Comment:
What public code are you referring to? (your comment points to line 56 which hardly has any code except for the  "const (").

If you are talking about the head constant in line 57, it is not public, it is private (its first letter is lowercase).  Also, the tests are in the same package (`ann`) as the rest of the code in this patch, meaning they can access private elements without the need of making them public.